### PR TITLE
User model

### PR DESCRIPTION
**Pull request by ermin-cahtarevic**
- Added the User model and validations
- Added the Post model and validations
- Tested the associations between User and Post models
- Added the Comment model and validations
- Tested the associations between User, Post, and Comment models

### CODE REVIEW

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi there team :hand: 
Great job on this project. But before moving on, we need to fix one thing.

- [ ] We need to add a descriptive readme file to our project.
Screenshot from 2019-12-20 08-52-49

Please submit for code review again after this change.
Happy coding :computer: 
**Sergio Torres**

File: README.md [hunk 1]
```diff
@@ -1,24 +1,36 @@
...
+# Micro Reddit
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> A simple Ruby on Rails App - Building with Active Record.
 
-Things you may want to cover:
 
-* Ruby version
+Simple app built for practice, working with rails models and validations.
```
Comment (samgaco) [inline README.md]: Try to explain the application more extensively, it's a good way of understanding the concepts better and the readme file would look cleared to visitors.

**Review by samgaco — APPROVED** [review]
Everything fine, you can move on now :+1: 

-------------------


* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

File: README.md (hunk 1 above)
Comment (ermin-cahtarevic) [reply README.md]: Explained the project in more detail as you proposed.

**Pull request merged**